src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinActivityConversionExtensions.cs

```diff
@@ -47,6 +47,46 @@ internal static ZipkinSpan ToZipkinSpan(this Activity activity, ZipkinEndpoint l
 
             activity.EnumerateTags(ref tagState);
 
+            if (activity.Status != ActivityStatusCode.Unset)
```

Comment:
add a comment above the `If` saying this code is executed when status is set on Activity using the native status field in activity, and not using activity tags.